CHANGELOG

```diff
@@ -1,6 +1,15 @@
 devel
 -----
 
+* APM-283: Use parallel gather in almost all queries. The only case were we
+  still cannot use parallel gather is when using traversals. All other queries
```

Comment:
This sounds rather pessimistic. According to the optimizer rule, we can actually parallelize the gather in some queries that use traversals.

Citing that code comment:
> we cannot parallelize the gather if we have a parallel traversal which
> itself depends again on a scatter/distribute node, because we are
> currently lacking synchronization for that scatter/distribute node.

We should probably use that some wording in the CHANGELOG to not sell the feature below its value.

Reply:
Unfortunately the situation is a bit more complicated... Generally we cannot parallelize when we have traversals. The exception is when we have a disjoint SmartGraph where the traversal can be performed completely on the local server. However, there is another exception for that case where the traversal is parallelized and somewhere above the traversal node we have another scatter/distribute, then even for the disjoint SmartGraph we _cannot_ parallelize. Will try to communicate this a bit better in the CHANGELOG.